frame_level_models.py

```diff
@@ -119,11 +109,32 @@ def create_model(self,
                    hidden_size=None,
                    is_training=True,
                    **unused_params):
+    """See base class.
+
+    Args:
+      model_input: A 'batch_size' x 'max_frames' x 'num_features' matrix of
+        input features.
+      vocab_size: The number of classes in the dataset.
+      num_frames: A vector of length 'batch' which indicates the number of
+        frames for each video (before padding).
+      iterations: the number of frames to be sampled.
+      add_batch_norm: whether to add batch norm during training.
+      sample_random_frames: whether to sample random frames or random sequences.
+      cluster_size: the output neuron number of the cluster layer.
+      hidden_size: the output neuron number of the hidden layer.
+      is_training: whether to build the graph in training mode.
+
+    Returns:
+      A dictionary with a tensor containing the probability predictions of the
+      model in the 'predictions' key. The dimensions of the tensor are
+      'batch_size' x 'num_classes'.
+    """
     iterations = iterations or FLAGS.iterations
     add_batch_norm = add_batch_norm or FLAGS.dbof_add_batch_norm
     random_frames = sample_random_frames or FLAGS.sample_random_frames
     cluster_size = cluster_size or FLAGS.dbof_cluster_size
     hidden1_size = hidden_size or FLAGS.dbof_hidden_size
+    act_fn = getattr(tf.nn, FLAGS.dbof_activation)
```

Comment:
I think it would be clearer to give the fully qualified name for the function as the argument and not assume it's in tf.nn.